### ISSUE TITLE
Feature Request: @JsonIdentityInfo processing should be disableable

### ISSUE THREAD

**Issue by dmitryDemchenko** [vojtechhabarta/typescript-generator#381]
FR #298 and commit 22b4fa6b90a44fbea3bfd5acf222a587cf944549 while using JSOGGenerator (https://github.com/jsog/jsog-jackson) like this 

``` java
import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.voodoodyne.jackson.jsog.JSOGGenerator;

@JsonIdentityInfo(generator=JSOGGenerator.class )
public class Issue {

    protected Product product;

}

@JsonIdentityInfo(generator=JSOGGenerator.class)
public class Product {

    protected String Title;

}
```
causes generated code for `Issue` to look like
``` typescript
export interface Issue {

        product: com.metalexpert.cms.model.Product | any; // | any - wrong!

}
```
that is not intended to be, as JSOG relinks all "serialized-by-ids" entities and types are strong as a result.
There should by some way to disable @JsonIdentityInfo processing.
We temporary had to rollback to 2.10.466 to avoid that.

**Comment by vojtechhabarta** [comment]
If I understand it correctly you want to generate TypeScript declarations for data returned from `JSOG.parse` method (as opposed to data returned directly from `JSON.parse`).

And best solution for you is some configuration switch which would disable `@JsonIdentityInfo` processing, right?

**Comment by dmitryDemchenko** [comment]
Exactly.
Thank you for fast response.

**Comment by vojtechhabarta** [comment]
I added `disableObjectIdentityFeature` parameter to `jackson2Configuration`. Here is Maven example:
``` xml
<configuration>
    <jsonLibrary>jackson2</jsonLibrary>
    <jackson2Configuration>
        <disableObjectIdentityFeature>true</disableObjectIdentityFeature>
    </jackson2Configuration>
</configuration>
```
It is released in [v2.16.538](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.16.538).

**Comment by dmitryDemchenko** [comment]
Thank you! I'll switch to v2.16.538 today.
